Amz.py

```diff
@@ -42,6 +41,39 @@ def pause_small():
     time.sleep(1)
 
 
+def give_away_type(driver):
+    try:
+        driver.find_element(By.XPATH, '//div[@class="amazon-video"]').click()
+        pause_small()
+        submit = WebDriverWait(driver, 20).until(
+            EC.element_to_be_clickable(
+                (By.XPATH, '//button[@class="a-button a-button-primary amazon-video-continue-button"]'))
+        )
+        submit.click()
+    except:
```

Comment:
It's not the best style/efficiency to throw all commands out there, and if something throws an error, then catch anything and throw all commands for the next type of execution.

A better/cleaner/more efficient method, would be be to figure out how to determine type of video to be played...if it's amz, do this, or YouTube, do that. This will help you moving forward to determine what errors occur and how to quickly fix them instead of "something happened, I guess". I realise this structure may not be an option for this situation, but give it a shot

Reply:
Ok I tried but idk if its the right way to do it.